### ISSUE TITLE
clang++ - error for cast time_t to Core::Time

### ISSUE THREAD

**Issue by gilcel** [SeisComP/common#153]
I get the following compile errors for casting `time_t` to `Core::TIme` with clang++ v16.0 on macOS.

```
src/base/common/libs/seiscomp/gui/datamodel/originlocatorview.cpp:6469:19: error: ambiguous conversion for functional-style cast from 'time_t' (aka 'long') to 'Core::Time'
 6469 |                 origin->setTime(Core::Time(dialog.getTime_t()));
      |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/base/common/libs/seiscomp/gui/datamodel/originlocatorview.cpp:6877:20: error: ambiguous conversion for functional-style cast from 'time_t' (aka 'long') to 'Time'
 6877 |                 derived->setTime(Time(dialog.getTime_t()));
      |                                  ^~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by gempa-jabe** [comment]
This has been fixed with 293dee7f6cd49601d09117bd5eedfe9e84e035e7.

**Comment by gilcel** [comment]
OK works! thanks!
